turbolinks/src/main/java/com/basecamp/turbolinks/TurbolinksJavascriptInjector.java

```diff
@@ -0,0 +1,74 @@
+package com.basecamp.turbolinks;
+
+import android.content.Context;
+import android.util.Base64;
+import android.webkit.ValueCallback;
+import android.webkit.WebView;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+public class TurbolinksJavascriptInjector {
```

Comment:
I made this its own class instead of exposing the entirety of `TurbolinksHelper` to the public domain.